fsr.ino

```diff
@@ -10,6 +10,8 @@
   #define SET_BIT(sfr, bit) (_SFR_BYTE(sfr) |= _BV(bit))
 #endif
 
+// Uncomment this line to make it so the built-in LED blinks when input is detected (if your Arduino/Teensy has a built-in LED...
```

Comment:
(Here and below) Can we use 80 line length limit? That's what I've been trying to follow in this file.